share/rocm/cmake/ROCMCreatePackage.cmake

```diff
@@ -315,6 +315,8 @@ macro(rocm_create_package)
 endmacro()
 
 macro(rocm_setup_license HEADER_ONLY)
+    set(CPACK_RPM_PACKAGE_LICENSE "MIT")
```

Comment:
This could set `CPACK_RPM_PACKAGE_LICENSE` to MIT by default, but it needs to be configurable per project. It should also be contained within the `if` block starting on line 345, so that if the license file is not detected or set, then no variables are set.

Reply:
Does it make sense to set a default license string? I guess this pull request is more of throwing the idea out there.

I can wrap this in a "if CPACK_RPM_PACKAGE_LICENSE is not defined" if that makes sense to you.

Reply:
Setting a default string would be convenient, but I could see it causing issues as it potentially obscures a mislabelling of the license in RPM packages (i.e. the license in the RPM would need to be checked against the license in the package to see issues where the variable wasn't correctly set, while with no default it's apparent because the license is 'unknown'). 

I'm torn though, as almost all projects use the MIT license, so having the default set makes a lot of sense and would save a lot of code duplication and PRs.